src/WorkBC.Data/JobBoardContext.cs

```diff
@@ -139,9 +139,16 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             // indexes for sorting job seekers on the admin screen
 
             // Status
-            modelBuilder.Entity<JobSeeker>()
-                .HasIndex(x => new { x.AccountStatus, x.LastName, x.FirstName })
-                .IncludeProperties(x => new { x.Email });
+            /*    modelBuilder.Entity<JobSeeker>()
+                   .HasIndex(x => new { x.AccountStatus, x.LastName, x.FirstName })
+                   .IncludeProperties(modelBuilder.Entity<>(x => new { x.Email }));
+
+
+                *
+                * NpgsqlIndexBuilderExtensions.IncludeProperties(
+       modelBuilder.Entity<Blog>().HasIndex(f => f.Name),
+       f => f.Description);
+               
```

Comment:
If we add a Postgres DB, the JobBoardContext would need to be modified to clarify which database is being used for the connection.